Updating securing apps and authorization documentation based on the adapter deprecation/removal plan (#21177)
keycloak/keycloak: docs/documentation/authorization_services/topics/enforcer-js-adapter.adoc

----

> Once you do that, you can create a `KeycloakAuthorization` instance as follows:

Reviewer: ```suggestion
Next, you can create a `KeycloakAuthorization` instance as follows:
```
